public/app/features/explore/ErrorContainer.tsx

```diff
@@ -11,7 +11,7 @@ export const ErrorContainer: FunctionComponent<ErrorContainerProps> = (props) =>
   const { queryError } = props;
   const showError = queryError ? true : false;
   const duration = showError ? 100 : 10;
-  const message = queryError ? queryError.message : null;
+  const message = queryError ? queryError.message ?? queryError.data?.message : null;
```

Comment:
Looks like the alert body is always shown, in that case, why not something like this?

```suggestion
  const message = queryError?.message || queryError?.data?.message || 'Unknown error';
```